pkg/scheduler/objects/queue.go

```diff
@@ -1823,3 +1853,10 @@ func (sq *Queue) recalculatePriority() int32 {
 	sq.currentPriority = curr
 	return priorityValueByPolicy(sq.priorityPolicy, sq.priorityOffset, curr)
 }
+
+// clearSortedQueues remove sortedQueue property. This is used for child queue get into stopped state.
+func (sq *Queue) clearSortedQueues() {
```

Comment:
I believe that this should be called all the way up to root. If a leaf is stopped, then it affects the entire hierarchy which is above, not just the immediate parent.